test/sharness/t0250-files-api.sh

```diff
@@ -597,9 +597,29 @@ test_files_api() {
     ipfs files ls /adir | grep foobar
   '
 
+  test_expect_failure "should fail to write file and create intermediate directories with no --parents flag set $EXTRA" '
+    echo "ipfs rocks" | ipfs files write --create /parents/foo/ipfs.txt
+  '
```

Comment:
This is not the correct way to do this.  Use `test_must_fail`.  Some thing like:
```
test_expect_success "should fail to write file and create intermediate directories with no --parents flag set $EXTRA" '
  echo "ipfs rocks" | test_must_fail ipfs files write --create /parents/foo/ipfs.txt
'
```
`test_expect_failure` is for broken test that need eventually be fixed.

Reply:
Fixed.